assets/admin/editor-wizard/steps/lesson-patterns-step.js

```diff
@@ -15,21 +19,67 @@ import PatternsStep from './patterns-step';
  * @param {Object} props.data Wizard data.
  */
 const LessonPatternsStep = ( { data, ...props } ) => {
+	const { senseiProExtension } = useSelect(
+		( select ) => ( {
+			senseiProExtension: select(
+				EXTENSIONS_STORE
+			).getSenseiProExtension(),
+		} ),
+		[]
+	);
+
 	const replaces = {};
 
 	if ( data.lessonTitle ) {
 		replaces[ 'sensei-content-title' ] = data.lessonTitle;
 	}
 
+	const isSenseiProInstalled =
+		! senseiProExtension || senseiProExtension.is_installed === true;
+
 	return (
-		<PatternsStep
-			title={ __( 'Lesson Type', 'sensei-lms' ) }
-			replaces={ replaces }
-			{ ...props }
-		/>
+		<Fragment>
+			<PatternsStep
+				title={ __( 'Lesson Patterns', 'sensei-lms' ) }
```

Comment:
Nice catch!

Reply:
Hey @aaronfc! Thanks for the review! 

Unfortunately, the title shouldn't be changed because the Figma files aren't updated according to what was decided on Slack. More context (and the fix) on the commit [34fdc93](https://github.com/Automattic/sensei/pull/5254/commits/34fdc93f0025cadabdc16d656aca3f2c29dc3acf)